lib/rex/logging/log_dispatcher.rb

```diff
@@ -147,10 +178,6 @@ def ilog(msg, src = 'core', level = 0)
 end
 
 def rlog(msg, src = 'core', level = 0)
-  if (msg == ExceptionCallStack)
-    msg = "\nCall stack:\n" + $@.join("\n") + "\n"
-  end
```

Comment:
Is `ExceptionCallStack` used anymore?

Reply:
No usages of the [variable or it's value](https://github.com/rapid7/metasploit-framework/blob/0d322a59711348ed94480b1405bc6a188c912f2b/lib/rex/logging/log_dispatcher.rb#L131) in the code base, plus no passing of any `msg` value to `rlog` in the code base that could trigger the above `if`. I'd say it's good to delete

Reply:
No usage in Pro that I can see, though there is some other `Rex` usage: https://github.com/search?q=org%3Arapid7+ExceptionCallStack&type=Code

Reply:
Un-deleted `ExceptionCallStack ` to prevent any potential issues with Rex gems moving forward